ci-build.sh

```diff
@@ -48,6 +48,9 @@ function start_test() {
 
 clean_up
 
+echo "Running Shellcheck against go.sh..."
+docker run --rm -v "$PWD:/mnt" koalaman/shellcheck:stable -x go.sh
```

Comment:
I'm not sure this is working quite as expected? If I check the CI logs I can this runs but the output isn't printed https://build.ci.pymnt.uk/job/pay-nginx-proxy/job/PR-39/3/console. If I run this locally it exists 1 so CI should fail right? 
```
gds5062-2:pay-nginx-proxy danworth$ docker run --rm -v "$PWD:/mnt" koalaman/shellcheck:stable -x go.sh
Unable to find image 'koalaman/shellcheck:stable' locally
stable: Pulling from koalaman/shellcheck
3ddc77a5eea0: Pull complete
b41a66f4a642: Pull complete
Digest: sha256:bc09eaa591263367f2531eaabafcbeaaf56f6f87aa4c0f736c40bbbbf6f33ff7
Status: Downloaded newer image for koalaman/shellcheck:stable

In go.sh line 9:
  dir=`mktemp -d`
      ^---------^ SC2006: Use $(...) notation instead of legacy backticked `...`.

Did you mean:
  dir=$(mktemp -d)


In go.sh line 56:
IFS=',' read -a LOCATIONS_ARRAY <<< "$LOCATIONS_CSV"
        ^--^ SC2162: read without -r will mangle backslashes.


In go.sh line 58:
    /enable_location.sh $((${i} + 1)) ${LOCATIONS_ARRAY[$i]}
                           ^--^ SC2004: $/${} is unnecessary on arithmetic variables.
                                      ^--------------------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean:
    /enable_location.sh $((${i} + 1)) "${LOCATIONS_ARRAY[$i]}"


In go.sh line 85:
    echo ${ADD_NGINX_SERVER_CFG}>/etc/nginx/conf/nginx_server_extras.conf
         ^---------------------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean:
    echo "${ADD_NGINX_SERVER_CFG}">/etc/nginx/conf/nginx_server_extras.conf


In go.sh line 94:
    proxy_cache_key "$scheme$request_method$host$request_uri";
                     ^-----^ SC2154: scheme is referenced but not assigned.
                            ^-------------^ SC2154: request_method is referenced but not assigned.
                                           ^---^ SC2154: host is referenced but not assigned.
                                                ^----------^ SC2154: request_uri is referenced but not assigned.

For more information:
  https://www.shellcheck.net/wiki/SC2154 -- host is referenced but not assign...
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
  https://www.shellcheck.net/wiki/SC2162 -- read without -r will mangle backs...
gds5062-2:pay-nginx-proxy danworth$ echo $?
1
```

Reply:
When I run it locally, it exits with a [1] status code. Which is what I believe causes the CI build script to fail.